app/pdfs/passengers_pdf.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
+  # what is the name of this file?
+  def initialize(passengers, filters)
+    super(page_layout: :landscape, page_size: "TABLOID" )
+    header(filters)
+    passengers_table(passengers)
+  end
+
+  def passengers_table(passengers)
+    font_size 14
+    passenger_table = [["Name", "Mobility Device ", "Phone", "Expiration Date", "Notes"]]
```

Comment:
Maybe I'm being silly, but `"Mobility Device "` probably doesn't need the space after it.